web_app/dist/main.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="icon" type="images/png" href="images/icon.png" />       
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>TaskViewer</title>
+        <meta name="description" content="Schedule tasks, without the clutter">
+        <link rel="stylesheet" href="main.css">
+        
+    </head>
+ 
+    <body>  
+        
+    <div id="sticky-header">
+        <div class="navbar">
+            <img id="logo" src="images/icon_small.png" alt="TaskViewer Logo">
+            <a>Log out</a>
+            <a>Sort</a>
+            <a>Filter by</a>
+            <input id="search-bar" type="text" placeholder="Search..">
+        </div>
+        <div class="new-flex-row-anchor-right">
+            <button id="create-task-button"class="button">+</button>    
+        </div>
+        <div class="new-flex-row-anchor-right">
+            <button id="delete-task-button" class="button">x</button>    
+        </div>
+    </div>    
+
+    
+    <div id="content">
+        <div class="task">
+            <div class="task-description-container">
+                <p class="task-text-spacing"><strong>Insert Task Here</strong></p>
```

Comment:
The comment I made earlier got hidden by some other change, but don't use `strong` here, use a class that makes it bold. In this case I'd say `task-name` just to be consistent with the naming scheme elsewhere.

In my opinion, we should have styling done in CSS, and structure done in HTML. Putting the style in the HTML itself makes the code inconsistent. As an added negative, manipulating boldness using JavaScript is more difficult when we use `strong` tags: it's easier to just select an element using JS and then add/ remove a class, compared to removing the `strong` tag without removing the text inside it, or selecting a specific bit of text and then adding `strong` around it.